src/sync/installation.ts

```diff
@@ -296,14 +296,21 @@ async function doProcessInstallation(app, data: BackfillMessagePayload, sentry:
 			try {
 				switch (task) {
 					case "build":
-						await jiraClient.workflow.submit(taskPayload.jiraPayload);
+						await jiraClient.workflow.submit(taskPayload.jiraPayload, {
+							preventTransitions: true,
+							operationType: "BACKFILL"
+						});
 						break;
 					case "deployment":
-						await jiraClient.deployment.submit(taskPayload.jiraPayload);
+						await jiraClient.deployment.submit(taskPayload.jiraPayload, {
+							preventTransitions: true,
+							operationType: "BACKFILL"
+						});
 						break;
 					default:
 						await jiraClient.devinfo.repository.update(taskPayload.jiraPayload, {
-							preventTransitions: true
+							preventTransitions: true,
+							operationType: "BACKFILL"
 						});
```

Comment:
From what I could find, this is where backfills requests happen. Please let me know if the options should be changed somewhere else.

Reply:
The above two jiraClient calls are for backfill too
`await jiraClient.workflow.submit(taskPayload.jiraPayload);`
`await jiraClient.deployment.submit(taskPayload.jiraPayload);`